python/strings_udf/strings_udf/__init__.py

```diff
@@ -41,59 +28,62 @@ def _get_appropriate_file(sms, cc):
         return None
 
 
+def maybe_patch_numba_linker(driver_version):
+    # Numba thinks cubinkinker is only needed if the driver is older than the ctk
+    # but when strings_udf is present, it might also need to patch because the PTX
+    # file strings_udf relies on may be newer than the driver as well
+    if driver_version < strings_udf_ptx_version:
+        logger.debug("Driver version %s.%s needs patching" % driver_version)
```

Comment:
It would be good if this mentioned that this was due to `strings_udf`, so that it's clear why the patch is being applied.